memote/suite/tests/test_basic.py

```diff
@@ -446,3 +418,53 @@ def test_find_unique_metabolites(read_only_model):
         total of {} ({:.2%}) unique metabolites in the model: {}""".format(
             len(ann["data"]), ann["metric"], truncate(ann["data"])))
     assert len(ann["data"]) < len(read_only_model.metabolites), ann["message"]
+
+
+@annotate(title="Number of Unique Metabolites", type="count")
```

Comment:
Remember to change the title to reflect the purpose/ output of the test!

Reply:
Oh and please remember to make place this test in the appropriate section in the test_config.yml!!